src/lib/date.ts

```diff
@@ -261,6 +309,62 @@ export function cascadingFormattedStartDateTime(
   return dateRange(startAt, endAt, timezone, "long")
 }
 
+export function auctionsDetailFormattedStartDateTime(startAt, endAt, endedAt) {
+  const thisMoment = moment.tz(moment(), "GMT")
```

Comment:
Is it right to hardcode "GMT" like that? (could follow the other patterns where it falls back otherwise uses the request's time zone).

Reply:
I thought about that, but I remembered from the design meetings that they specifically wanted GMT for that and then you could mouse over a little question mark and it would display local, happy to have this follow the rest though

Reply:
vaguely remember that! cool